README.md

```diff
@@ -127,127 +127,213 @@ future state of the libraries and to discover projects for contribution.
+the hub for conversations, community signal (+1s) and the issue number is used
+as the unique identifier of this RFC.
+
+> Before creating a tracking issue, please search for similar or related ideas in
+the RFC table above or in the issue list of this repo. If there is a relevant
+RFC, collaborate on that existing RFC, based on its current stage.
+
+Our [tracking issue template] includes a checklist of all the steps an RFC goes
+through and it's the driver's responsibility to update the checklist and assign
+the correct label to on the RFC throughout the process.
+
+[tracking issue template]: https://github.com/aws/aws-cdk-rfcs/blob/master/.github/ISSUE_TEMPLATE/tracking-issue.md
+
+When the issue is created, it is required to fill in the following information:
+
+1. **Title**: the name of the feature or change - think changelog entry.
+2. **Description**: a _short_ description of feature, as if it was already implemented.
+3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
+   under "Proposed by".
+
+### 2. API Bar Raiser
+
+Reach us via [#aws-cdk-rfcs] to get an "API Bar Raiser" assigned to your RFC.
+
+For each RFC, CDK leadership will assign an **API Bar Raiser** who reviews and
+approves the public API of the feature. API Bar Raisers have veto rights on
+API-related design decisions, such as naming, structure, options, CLI commands
+and others.
+
+The public API of a feature represents the surface through which users interact
+with it, and we want to make sure these APIs are consistent, ergonomic and
+designed based on the intent and the mental model of our users. Additionally,
+once we announce that a feature is "stable" (1.0, GA, etc) any breaking change
+to its public API will require releasing a new major version, so we like think
+of API decisions as "one way doors".
+
+API Bar Raisers will be assigned using a tiering model which is generally based
+on the size of the user base that will likely get exposed to the feature. As a
+general rule, the more "significant" the feature is, we will assign a bar raiser
+with a wider and longer-term context of the project.
+
+To merge an RFC, a [sign-off](#6-api-sign-off) from the bar raiser is required
+on the public API of the feature, so we encourage to engage with them early in
+the process to make sure you are aligned on how the API should be designed.
+
+> NOTE: The technical solution proposed in an RFC *does not* require approval
+> beyond the normal pull request approval model (e.g. a core team member needs
+> to approve the RFC PR and any subsequent changes to it).
+
+### 3. Kick-off
+
+Before diving into writing the RFC, it is highly recommended to organize a
+kick-off meeting that includes the API Bar Raiser and any stakeholders that
+might be interested in this RFC or can contribute ideas and direction. The goal
+of the meeting is to discuss the feature, its scope and general direction for
+implementation.
```

Comment:
Perhaps suggest how to go about this? From an external collaborator's perspective, it is not clear how to get a hold of people's calendars, what conferencing solutions are available - which are free to use? which are acceptable?

Reply:
Good call. I'll add a suggestion to reach out to the team via the #aws-cdk-rfcs slack channel to get help organizing this

Reply:
Added